INSTALL.rst

```diff
@@ -1,64 +1,509 @@
 Installing scikit-image
 =======================
 
+We outline 4 different ways that we recommend to get started with image
+processing and scikit-image.
+
+1. `Scientific Python Distributions`_ -- Recommended
+2. `pip based installation`_
+3. `conda based installation`_
+4. `System-wide package manager`_
+
+After your installation, you should be able to run the following code
+from a Python console or Jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+For those attempting to build scikit-image from source, or trying contribute
+code back to scikit-image, we've provided instructions under the heading
+`Installing scikit-image for contributors`_.
+
+Scientific Python Distributions
+-------------------------------
+
+``scikit-image`` comes pre-installed with several Python distributions.
+These have the advantage of comming pre-installed with many other useful
```

Comment:
Just a small misspell: change `comming` to `coming`.

Reply:
thanks!